content/docs/hooks/use-slider.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: useSlider
+package: '@chakra-ui/slider'
+description: 'React hook to manage sliders.'
+---
+
+`useSlider` is a custom hook used to provide slider functionality, as well as
+state and focus management to custom slider components when using it.
+
+## Import
+
+```js
+import { useSlider } from '@chakra-ui/react'
+```
+
+## Return value
+
+The `useSlider` hook returns following props
```

Comment:
It'll be helpful to include 'the' in this sentence, so it reads: "The useSlider hook returns **the** following props"

Reply:
Great work @Tisks! There's a conflict with the PR that needs resolving. Everything else looks good.